### PR TITLE
[RFR] Fix migrations after repo merge

### PR DESCRIPTION
**Pull request by cwisecarver**
## Purpose

Updates migrations to work with new table naming scheme and moves missing changes from osf_models to osf.io.

## Changes

Updates migrations with correct table names, move NodeRelation migration from osf_models repo to osf.io. Add necessary fields to analytics models for migration.

### CODE REVIEW

**Comment by cwisecarver** [comment]
@sloria This should fix node relation migrations and let us give osf_models repo a viking funeral.

File: osf/management/commands/migraterelations.py [hunk 1]
```diff
@@ -238,9 +246,9 @@ def format_lookup_string(modm_obj):
                                     continue
                                 if value.__class__.__name__ in ['Node', 'Registration', 'Collection',
                                                                 'Preprint']:
-                                    gfk_model = apps.get_model('osf', 'AbstractNode')
+                                    gfk_model = apps.get_model('osf_models', 'AbstractNode')
```
Comment (sloria) [inline osf/management/commands/migraterelations.py]: Why was this changed?

**Comment by sloria** [comment]
Looks good to merge; just the one issue noted above.

File: osf/management/commands/migraterelations.py (hunk 1 above)
Comment (cwisecarver) [inline osf/management/commands/migraterelations.py]: I think I missed it in the rebase. Grabbed the wrong line. I'll change it.

**Comment by cwisecarver** [comment]
Fixed, merging.
